src/solvers/refinement/string_refinement.cpp

```diff
@@ -119,15 +119,16 @@ static std::vector<exprt> instantiate(
 static optionalt<exprt> get_array(
   const std::function<exprt(const exprt &)> &super_get,
   const namespacet &ns,
-  const std::size_t max_string_length,
+  std::size_t max_string_length,
```

Comment:
Unclear why some parameters are const and others aren't.

Reply:
If the parameter is not a reference, `const` only has an effect at the function definition and not at declaration.